protected/humhub/modules/user/models/forms/AccountChangeUsername.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+	/**
+	 * @link https://www.humhub.org/
+	 * @copyright Copyright (c) 2016 HumHub GmbH & Co. KG
+	 * @license https://www.humhub.com/licences
+	 */
+
+	namespace humhub\modules\user\models\forms;
+
+	use Yii;
+	use yii\helpers\Url;
+	use humhub\modules\user\models\User;
+	use humhub\modules\user\components\CheckPasswordValidator;
+	use humhub\modules\user\components\UsernameValidator;
+
+	/**
+	 * Form Model for username change
+	 *
+	 * @since 0.5
+	 */
+	class AccountChangeUsername extends \yii\base\Model
+	{
+
+		/**
+		 * @var string the users password
+		 */
+		public $currentPassword;
+
+		/**
+		 * @var string the users new username
+		 */
+		public $newUsername;
+
+		/**
+		 * @inheritdoc
+		 */
+		public function rules()
+		{
+			/* @var $userModule \humhub\modules\user\Module */
+        	$userModule = Yii::$app->getModule('user');
+
+			$rules = [
```

Comment:
The validation rules are redundant, either extend this class from the User Model and make use of scenarios or add all validations to your UsernameValidator.